system/API/ResponseTrait.php

```diff
@@ -102,6 +102,13 @@ trait ResponseTrait
 	 */
 	protected $format = 'json';
 
+	/**
+	 * Current Formatter instance. This is usually set by ResponseTrait::format
```

Comment:
We should make sure this plays nicely with the current JSON API pull request, in case that one goes through. #3545

Reply:
I don't get how this will conflict with that PR. It tackles the `respond` method which does not use `$this->formatter` directly. Also, the property is uninitialized so classes using the trait will not have problems assigning a value to that.

Reply:
Okay great. That was my hunch but just wanted to make sure!